ChatSecure/Classes/View Controllers/OTRMessagesViewController.h

```diff
@@ -64,4 +72,7 @@
 /** override this method to customize what should be shown at the beginning of the message status */
 - (nullable NSAttributedString *) encryptionStatusStringForMessage:(nonnull id<OTRMessageProtocol>)message;
 
+/** Override this to return a fallback conference service JID. Will only be called if no conference server was discovered. Return nil to abort group chat creation. May be used to show warning message to user. */
+- (nullable NSString *)getFallbackConferenceServiceJID;
```

Comment:
This shouldn't really be in here. I'd prefer not to have any reference to this hack upstream at all. I really don't agree with using a client hack to work around a trivial server configuration issue.